### PR TITLE
Map reading from map.txt file.

### PR DESCRIPTION
**Pull request by dejancesarov-playrix**
Loading map from map.txt, showing only wall, ignoring player and enemies.
Printing map on Render call.

### CODE REVIEW

File: src/Map.cpp [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+	{
+		for(auto& it : line)
+		{
+			if (it != wall)
+			{
+				it = empty;
+			}
+		}
+
+		_grid.push_back(std::move(line));
```
Comment (dusanslivka-playrix) [inline src/Map.cpp]: mislim da je bolje koristiti **emplace_back()** umesto **push_back()**.

File: src/Map.cpp [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+void Map::Load(const std::string& filePath)
+{
+	std::ifstream fileStream(filePath);
+	FillData(fileStream);
+}
+
+void Map::FillData(std::ifstream& stream)
+{
+	const auto wall = static_cast<char>(PointData::Wall);
+	const auto empty = static_cast<char>(PointData::Empty);
```
Comment (dusanslivka-playrix) [inline src/Map.cpp]: Ne treba korisiti auto za primitivne tipove. [(link)](https://knowledge.playrix.com/display/development/Coding+Standards#CodingStandards-auto)

File: src/Map.cpp [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+}
+
+void Map::FillData(std::ifstream& stream)
+{
+	const char wall = static_cast<char>(PointData::Wall);
+	const char empty = static_cast<char>(PointData::Empty);
+
+	for (std::string line; std::getline(stream, line);)
+	{
+		for(auto& it : line)
```
Comment (igorkukobat-playrix) [inline src/Map.cpp]: Space ti fali

File: src/Map.cpp [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+			}
+		}
+
+		_grid.emplace_back(std::move(line));
+	}
+}
+
+void Map::Render()
+{
+	system("cls");
```
Comment (leventesimonplayrix) [inline src/Map.cpp]: svaki put se obrise ceo ekran? i ako je neko drugi crtao nesto?

**Pull request merged**